BeastXUD.md

```diff
@@ -29,17 +29,11 @@ Two options are available:
 
 ## Setup
 
-1. [Download Ubuntu Server](https://ubuntu.com/download/server) onto your computer. Any other linux distribution supporting docker is also fine.
+1. [Download Ubuntu Server 20.04 LTS](https://ubuntu.com/download/server) onto your computer. Any other linux distribution supporting docker is also fine. This guide was written assuming `Ubuntu Server 20.04 LTS`.
 2. Insert a USB Stick into your computer and [create the a bootable USB Stick](https://ubuntu.com/tutorials/tutorial-create-a-usb-stick-on-ubuntu) with the ubuntu image you just downloaded.
 3. Open your Mini PC, plug in RAM & drives, close it, connect it to your router via ethernet cable and to a power supply. Connect a screen via HDMI, a USB keyboard, the created bootable USB Stick, fire the Mini PC up and follow the the inital setup instructions.
 4. Update ubuntu via `sudo apt update && sudo apt upgrade`
-5. Install docker following the [official instructions](https://docs.docker.com/install/linux/docker-ce/ubuntu/). At the time of writing, ubuntu 19.10 was still not supported in the official docker repos. If you see an error along the lines `'docker-ce' has no installation candidate` in step 4 of the docker instructions, run this instead:
-```bash
-sudo add-apt-repository \
-   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
-   disco \
-   stable"
-```
+5. Install docker by running `sudo apt install docker.io`. If you are not using Ubuntu 20.04 LTS, follow the [official instructions](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
```

Comment:
FYI, and I have run into this locally since upgrading to 20.04, there is no ppa for docker in ubuntu 20.04 and the official install instructions don't support it. From the link here you'll see it's not included (yet):

> To install Docker Engine, you need the 64-bit version of one of these Ubuntu versions:
> 
>     Ubuntu Eoan 19.10
>     Ubuntu Bionic 18.04 (LTS)
>     Ubuntu Xenial 16.04 (LTS)

Either way, I think it might make sense not to specify exact ubuntu versions here (and a few lines above) since official support doesn't appear to be there and since versions will change over time.

Reply:
> Install docker by running `sudo apt install docker.io`

This installs the ubuntu-maintained version, relatively up-to-date (`19.03.8`) and is the only way to install docker properly on ubuntu 20.04. I tested this on multiple machines and it's working fine.

The funny thing about the docker official instructions is, that they are wrong (e.g. ubuntu `19.10` is actually **not** supported, but `19.04` is, so you manually have to their install script to get things working. But since I don't really want to take care of fixing docker install instructions and things work out of the box on 20.04, that's what I want to recommend:
- if you run ubuntu 20.04, install docker with `sudo apt install docker.io`
- if you run something else, follow the instructions: https://docs.docker.com/install

Any idea how to express it better?

Reply:
Gotcha, thanks for the explanation. How about:

> If you are using Ubuntu 20.04 LTS, install docker by running `sudo apt install docker.io`. Otherwise if you are using any version besides 20.04, follow the [official instructions](https://docs.docker.com/install/linux/docker-ce/ubuntu/) to install docker.

Reply:
Done, thx!